src/lib/viewers/doc/_docBase.scss

```diff
@@ -348,6 +348,12 @@ $thumbnail-sidebar-width: 226px;
                     pointer-events: none;
                 }
             }
+
+            .annotationLayer {
+                section {
+                    pointer-events: none;
+                }
+            }
```

Comment:
Is `section` needed? Can we ignore the whole layer?
```
.annotationLayer {
    pointer-event: none;
}
```

Reply:
We can ignore the whole layer.